youtube_bz/main.py

```diff
@@ -43,6 +44,8 @@ def get_command_parser() -> argparse.ArgumentParser:
     download_parser.add_argument("mbid", help="music brainz identifer of a release")
     download_parser.add_argument("--verbose", action="store_true")
 
+    download_parser.add_argument("--destination", help="Path to the output directory")
```

Comment:
Maybe we can add a short flag like `-d` too.
e. g `download_parser.add_argument("-d", "--destination", help='Path to the output directory")`

Reply:
Sure thing!